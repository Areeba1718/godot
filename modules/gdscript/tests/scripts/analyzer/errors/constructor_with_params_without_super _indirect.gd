# https://github.com/godotengine/godot/issues/40861

func test():
	var _sub = SubSubClass.new()

class SuperClass:
	func _init(param):
		print(param)

class SubClass extends SuperClass:
	# No custom constructor.
	pass

class SubSubClass extends SubClass:
	func _init():
		print("Missing super()")
