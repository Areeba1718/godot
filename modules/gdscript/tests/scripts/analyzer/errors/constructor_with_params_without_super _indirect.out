GDTEST_ANALYZER_ERROR
Constructor must call "super()" to invoke the superclass constructor.
